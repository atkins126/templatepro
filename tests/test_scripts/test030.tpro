{{loop(customers) as cust}}
X
{{endloop}}

HEADER
{{reset(customers)}}
{{# this is a comment }}
{{loop(customers) as cust}}
{{# this is a 
	multi line comment }}
ABCD{{:cust.Code}}EFG{{:cust.Name}}
{{loop(objects) as obj}}
ABCD{{:obj.Prop1}}EFG{{:obj.Prop2}}
{{endloop}}
{{endloop}}