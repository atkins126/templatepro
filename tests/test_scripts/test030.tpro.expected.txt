X
X
X
X
X
X
X
X
X
X
X
X
X

HEADER


ABCD1EFGFord
ABCDvalue1.1EFGvalue2.1
ABCDvalue1.2EFGvalue2.2
ABCDvalue1.3EFGvalue2.3

ABCD2EFGFerrari

ABCD3EFGLotus

ABCD4EFGFCA

ABCD5EFGHyundai

ABCD6EFGDe Tomaso

ABCD7EFGDodge

ABCD8EFGTesla

ABCD9EFGKia

ABCD10EFGTata

ABCD11EFGVolkswagen

ABCD12EFGAudi

ABCD13EFGSkoda
